gradle.properties

```diff
@@ -80,3 +80,4 @@ cred_path=/home/ubuntu
 config_path=/home/ubuntu
 pravega_org=pravega
 pravega_branch=master
+travis_commit=0
```

Comment:
travis_commit is populated in the `./aws/postTestScipt.sh`, do we require it in gradle.properties?